src/main/app.ts

```diff
@@ -38,7 +38,9 @@ app.use((req, res, next) => {
     next();
   });
 app.use("/", RouterFinder.findAll(path.join(__dirname, "routes")));
-
+if(developmentMode){
```

Comment:
formatting looks weird here, missing spaces? is there linting here?

Reply:
There is linting , but looks like it doesn't work. Will fix it as bug in different PR.